tests/std/tests/P0323R12_expected/test.cpp

```diff
@@ -2135,6 +2136,11 @@ void test_lwg_3843() {
     }
 }
 
+void test_gh_4011() {

```

Comment:
1. We like to have a comment describing what problem a regression test is trying not to regress for the day the bug database inevitably goes away.
2. This doesn't need to be run, or even in a function; we could just put these `static_assert`s at namespace scope.